src/signTx.c

```diff
@@ -250,6 +313,10 @@ static void signTx_handleInitAPDU(uint8_t p2, uint8_t* wireDataBuffer, size_t wi
 		VALIDATE(ctx->numCertificates < SIGN_MAX_CERTIFICATES, ERR_INVALID_DATA);
 		VALIDATE(ctx->numWithdrawals < SIGN_MAX_REWARD_WITHDRAWALS, ERR_INVALID_DATA);
 
+		// TODO isn't this more like a security policy?
+		if (ctx->isSigningPoolRegistrationAsOwner)
+			VALIDATE(ctx->numCertificates == 1, ERR_INVALID_DATA);
```

Comment:
I would expect here an enforcement of numWithdrawals being zero as well. How is this ensured right now?